lib/omnibus/packagers/rpm.rb

```diff
@@ -630,6 +630,8 @@ def safe_architecture
       case Ohai["kernel"]["machine"]
       when "i686"
         "i386"
+      when "armv7l" # raspberry pi 3 CentOS
+        "armv7hl"
       when "armv6l"
         if Ohai["platform"] == "pidora"
```

Comment:
do you need to replicate the same logic as down here?

Reply:
I don't have 'pidora', so I figured I'd leave it on a simple path until someone shows up with another 32-bit ARM RPM-based distro that needs support. I'm testing with CentOS 7.6.

Reply:
@lamont-granquist do you consider this a blocker, or can we merge it?